src/Elastic.OpenTelemetry/Extensions/ResourceBuilderExtensions.cs

```diff
@@ -70,9 +68,7 @@ internal static ResourceBuilder AddElasticResourceDefaults(this ResourceBuilder
 			.AddDistroAttributes()
 			.AddEnvironmentVariableDetector();
 
-#if NET462_OR_GREATER || NET6_0_OR_GREATER
```

Comment:
@stevejgordon it didn't seem to be necessary to guard these for netstandard?  

Let me know if I missed something 😸

Reply:
Yeah, I'm not sure why I did that now. Some of the `HostDetector` code is limited to specific targets, but we should add it in all cases. Let's keep your change and if I recall a reason we can switch it back!